_html/templates/my-profile-page.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+* My profile page - view mode.
+*/
+
+include "inc/header.inc.php";
+include "modules/header.php";
+?>
+
+<div class="section-intro sponsors-intro">
```

Comment:
Move all HTML to a different PHP file in modules folder and just include it here.